httpserver/http_connection.cpp

```diff
@@ -302,7 +304,7 @@ void connection_t::read_request() {
 bool connection_t::validate_snode_request() {
     if (!parse_header(LOKI_SENDER_SNODE_PUBKEY_HEADER,
                       LOKI_SNODE_SIGNATURE_HEADER)) {
-        LOKI_LOG(error, "Missing signature headers");
+        LOKI_LOG(debug, "Missing signature headers");
```

Comment:
Could we maybe make this `"Snode request missing signature headers"`?

Reply:
updated